src/utils/keychainConstants.js

```diff
@@ -0,0 +1,6 @@
+export const seedPhraseKey = 'rainbowSeedPhrase';
+export const privateKeyKey = 'rainbowPrivateKey';
```

Comment:
can we update redux/wallets.js line 299: const pkeyFound = await hasKey(`${account.address}_rainbowPrivateKey`); to use privateKeyKey